packages/js/product-editor/src/components/product-section-layout/product-section-layout.tsx

```diff
@@ -1,14 +1,20 @@
 /**
  * External dependencies
  */
-import { Children, isValidElement, createElement } from '@wordpress/element';
+import {
+	Children,
+	isValidElement,
+	createElement,
+	Element,
+} from '@wordpress/element';
 import { FormSection } from '@woocommerce/components';
 
-type ProductSectionLayoutProps = {
+interface ProductSectionLayoutProps {
 	title: string;
 	description: string | JSX.Element;
 	className?: string;
-};
+	children: Element[] | Element;
```

Comment:
Similar comment as above: https://github.com/woocommerce/woocommerce/pull/54312/files#r1910079027
Using `React.PropsWithChildren< ProductSectionLayoutProps >`?

Having said that, these components can also be removed as they are currently not used. But that can be a follow up.

Reply:
Fixed with 0822966434e46bd39dbbcddba65d7b2149c5092a